jQRangeSliderLabel.js

```diff
@@ -349,14 +349,18 @@
 			var label1Pos = this.GetRawPosition(this.cache.label1, this.cache.handle1),
 				label2Pos = this.GetRawPosition(this.cache.label2, this.cache.handle2);
 
-			if (this.label1[type]("option", "isLeft")){
-				this.ConstraintPositions(label1Pos, label2Pos);
-			}else{
-				this.ConstraintPositions(label2Pos, label1Pos);
+			if (this.label1[0].innerText === this.label2[0].innerText) {
```

Comment:
This change breaks the edit range slider which does not have any inner text. It should not rely on the text but on values to decide to merge labels.